### ISSUE TITLE
unable to gulp serve code from tutorial #2

### ISSUE THREAD

**Issue by dcu-sharepoint** [SharePoint/sp-dev-docs#348]
#### Category

- [X ] Question

#### Expected or Desired Behavior

the following code is expected to build without errors as per lab #2 

`private _getMockListData(): Promise<ISPLists>{
    return MockHttpClient.get(this.context.pageContext.web.absoluteUrl).then(() => {
      const listData: ISPLists = {
       value:
       [
        **{Title: 'Mock List One',   Id: '1'},
        {Title: 'Mock List Two',   Id: '2'},
        {Title: 'Mock List Three', Id: '3'}**
       ]
      };
      return listData;
    }) as Promise<ISPLists>;
  }`

#### Observed Behavior

![failingcode](https://cloud.githubusercontent.com/assets/22737071/21959156/e9efa90a-da8c-11e6-95b4-b802c41210fa.png)
![error-description](https://cloud.githubusercontent.com/assets/22737071/21959236/0ea26f2e-da8f-11e6-979a-04b6cdcb992f.png)

![console-ouput](https://cloud.githubusercontent.com/assets/22737071/21959159/f587cb12-da8c-11e6-8faf-2ec1a88e135b.png)

=> As you can see, the ISPLists variable can not be assigned as shown. this code is exactly as used in the lab #2 in the sequence of 6 labs this being **SharePoint Framework Tutorial 2 - HelloWorld, Talking to SharePoint** lab.

The question is:
**am I missing an import of some type** other than the import:

`import MockHttpClient from './MockHttpClient';`

// LCR Function to mimick data from a sharepoint list

`import {ISPList } from './HolaMundoWebPart';`

for the defined class:

`export default class MockHttpClient {

	private static _items: ISPList[] = [{Title: 'Mock List', Id: '1'}];

	public static get(resultUrl: string, options?: any): Promise<ISPList[]>{
		return new Promise<ISPList[]>((resolve) => {
			resolve(MockHttpClient._items);
		});
	}
}`

Thank your for your patience for newbee questions.

Thanks for your contribution! Sharing is caring.

**Comment by dcu-sharepoint** [comment]
found some typos, re-testing

**Comment by dcu-sharepoint** [comment]
I rechecked the code against the video and cleared a couple of typos but the result is still a bad gulp build for the code as shown.

**Comment by dcu-sharepoint** [comment]
I couln't get the code as shown in the images to work. however, I reworked the code in the functions and variable declarations until I got the project to build with the gulp serve command and the webpart loaded correctly int the local workbench.

would like to know, however, what is wrong with code as copied from the lab2 example. it works on the video but it sure as hell does not work when copied to a project.
